pkg/webhook/mutating/server.go

```diff
@@ -0,0 +1,102 @@
+package mutating
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/inject"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+const (
+	webhookName      = "nmstate"
+	RetrievalTimeout = 120
```

Comment:
I'd prefer not using a constant. But if we do, let's keep it with the unit (time.Second) and name it better (CARetrievalTimeout)?